elements/lisk-cryptography/src/hash.ts

```diff
@@ -43,3 +43,8 @@ export const hash = (data: Buffer | string, format?: string): Buffer => {
 		'Unsupported data format. Currently only Buffers or `hex` and `utf8` strings are supported.',
 	);
 };
+
+export const getNetworkIdentifier = (
```

Comment:
Could you add tests for checking behaviour when the inputs are `null`, `undefined` and `''` (empty string) and maybe consider adding validation to not allow any of those values? Of course in TS it's not a problem but when using the function from JS we can pass anything. 

Otherwise looks good to go!

Reply:
These types are already checked for in the `hash` function. The hash function will throw if the input is anything other than `string` or `Buffer`.